odk1-src/form-question-types.rst

```diff
@@ -1054,6 +1054,45 @@ you can add :tc:`or_other`.
 This will add "Other" as an additional option to your choice list.
 The :th:`name` value of the choice when selected will be :tc:`other`.
 
+
+.. _rank-widget:
+
+Rank widget
+-----------------
+
+The rank widget allows the user to order options from a list. The value saved in the form and sent to the server is a space-separated ordered list of the options.
+
+Like with :ref:`select-widgets`, the options are listed on a sheet named **choices** in an XLSForm.
+
+To change the order of the options in the list, tap the "Rank items" button. In the resulting dialog, long press on items and once they get a border around them, drag them up or down to change their order. If no :ref:`default <default-responses>` is provided, the value for the question is blank until the user taps "OK" in the ranking dialog.
```

Comment:
s|"Rank items"|:guilabel:`Rank items`|
s|"OK"|:guilabel:`OK`|

Long press is on a single item, not items. So...
> long press on an item and once it gets a border around it, drag it up or down to change the order.